packages/lib/src/components/Redirect/components/RedirectShopper/RedirectShopper.tsx

```diff
@@ -49,14 +51,17 @@ class RedirectShopper extends Component<RedirectShopperProps> {
                 })
         );
 
-        dispatchEvent.then(doRedirect).catch(() => {});
+        dispatchEvent.then(doRedirect).catch(() => {
+            this.props.onRedirectError();
```

Comment:
Can you recall me in which scenarios does the Redirect may fail?

Reply:
To be honest, I cannot think of that many situations except for `beforeRedirect` callback gets rejected. `doRedirect` looks reliable to me so i cannot think of an error situation at this moment.